.changeset/flat-pandas-cross.md

```diff
@@ -0,0 +1,6 @@
+---
+"@apollo/federation-internals": patch
+---
+
+Fix fed1 schema upgraded when `@external` is on a type.
```

Comment:
I'm not sure a user reading this will understand how this affects their usage. I think the context in your PR description would be useful here (and in the manual changelog entries).

Reply:
You're right, this wasn't that helpful as it could be. I've reworded it so it's more obvious for a user if they may be affected or not by this and care about.